2022-02-10 23:10:03,884 INFO    MainThread:639856 [wandb_setup.py:_flush():71] setting env: {}
2022-02-10 23:10:03,884 INFO    MainThread:639856 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-10 23:10:03,884 INFO    MainThread:639856 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_231003-18auivqb/logs/debug.log
2022-02-10 23:10:03,884 INFO    MainThread:639856 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_231003-18auivqb/logs/debug-internal.log
2022-02-10 23:10:03,884 INFO    MainThread:639856 [wandb_init.py:init():404] calling init triggers
2022-02-10 23:10:03,884 INFO    MainThread:639856 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-10 23:10:03,884 INFO    MainThread:639856 [wandb_init.py:init():460] starting backend
2022-02-10 23:10:03,884 INFO    MainThread:639856 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-10 23:10:03,904 INFO    MainThread:639856 [backend.py:ensure_launched():216] starting backend process...
2022-02-10 23:10:03,928 INFO    MainThread:639856 [backend.py:ensure_launched():221] started backend process with pid: 640058
2022-02-10 23:10:03,934 INFO    MainThread:639856 [wandb_init.py:init():469] backend started and connected
2022-02-10 23:10:03,938 INFO    MainThread:639856 [wandb_init.py:init():533] updated telemetry
2022-02-10 23:10:04,010 INFO    MainThread:639856 [wandb_init.py:init():563] communicating current version
2022-02-10 23:10:04,861 INFO    MainThread:639856 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-10 23:10:04,861 INFO    MainThread:639856 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-10 23:10:04,980 INFO    MainThread:639856 [wandb_init.py:init():606] starting run threads in backend
2022-02-10 23:10:08,050 INFO    MainThread:639856 [wandb_run.py:_console_start():1810] atexit reg
2022-02-10 23:10:08,050 INFO    MainThread:639856 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-10 23:10:08,050 INFO    MainThread:639856 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-10 23:10:08,098 INFO    MainThread:639856 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-10 23:10:08,098 INFO    MainThread:639856 [wandb_init.py:init():633] run started, returning control to user process
2022-02-10 23:10:08,098 INFO    MainThread:639856 [wandb_run.py:_config_callback():956] config_cb None None {}
2022-02-10 23:10:11,314 INFO    MainThread:639856 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-10 23:10:11,318 INFO    MainThread:639856 [wandb_run.py:_restore():1752] restore
2022-02-10 23:10:13,617 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 250473
  total_bytes: 250473
}

2022-02-10 23:10:13,782 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 250473
  total_bytes: 250473
}

2022-02-10 23:10:14,005 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 250473
  total_bytes: 257268
}

2022-02-10 23:10:14,106 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 250473
  total_bytes: 264397
}

2022-02-10 23:10:14,208 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 256951
  total_bytes: 264397
}

2022-02-10 23:10:14,309 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 264397
  total_bytes: 264397
}

2022-02-10 23:10:14,411 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 264397
  total_bytes: 264397
}

2022-02-10 23:10:14,514 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 264397
  total_bytes: 264397
}

2022-02-10 23:10:14,615 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 264397
  total_bytes: 264397
}

2022-02-10 23:10:14,717 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 264397
  total_bytes: 264397
}

2022-02-10 23:10:14,906 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 264397
  total_bytes: 264397
}

2022-02-10 23:10:15,161 INFO    MainThread:639856 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
  media_count: 1
}
pusher_stats {
  uploaded_bytes: 264397
  total_bytes: 264397
}
local_info {
}

2022-02-10 23:10:16,297 INFO    MainThread:639856 [wandb_run.py:_append_history():2130] rendering history
2022-02-10 23:10:16,297 INFO    MainThread:639856 [wandb_run.py:_append_summary():2085] rendering summary
2022-02-10 23:10:16,297 INFO    MainThread:639856 [wandb_run.py:_append_files():2180] logging synced files
